package-lock.json

```diff
@@ -452,6 +453,11 @@
       "integrity": "sha512-k/vGaX4/Yla3WzyMCvTQOXYeIHvqOKtnqBduzTHpzpQZzAskKMhZ2K+EnBiSM9zGSoIFeMpXKxa4dYeZIQqewQ==",
       "dev": true
     },
+    "node_modules/lodash.debounce": {
```

Comment:
what about all of this? Do we need all of this? I didn't see any usage below

Reply:
ok